packages/cli-hydrogen/src/cli/commands/hydrogen/deploy.ts

```diff
@@ -34,6 +34,11 @@ export default class Deploy extends Command {
       env: 'SHOPIFY_HYDROGEN_FLAG_DMS_ADDRESS',
       default: 'https://oxygen-dms.shopifycloud.com',
     }),
+    skipHealthCheck: Flags.boolean({
+      env: 'SHOPIFY_HYDROGEN_FLAG_SKIP_HEALTH_CHECK',
+      default: false,
+      description: 'Exit directly after upload without waiting for webpage to be available',
```

Comment:
If you go with my above change, we can just change the copy of this description to also match what is currently on `oxygenctl`. Obviously Graham will be looking over all of this so not a big deal.

`Require a health check before the deployment succeeds.`